x-pack/plugins/security_solution/public/management/pages/policy/view/event_filters/list/policy_event_filters_list.test.tsx

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
Can you add tests that verify the correct call of APIs with and without filters applied?